src/sentry/issues/ingest.py

```diff
@@ -210,6 +210,7 @@ def send_issue_occurrence_to_eventstream(
         primary_hash=occurrence.fingerprint[0],
         received_timestamp=group_event.data.get("received") or group_event.datetime,
         skip_consume=False,
+        group_id=group_info.group.id,
```

Comment:
Why do we need to pass this now?

Reply:
I guess I just misread the GH issue details. Will remove